webv2/updateTableSchema/interleaved_test.go

```diff
@@ -0,0 +1,143 @@
+package updateTableSchema
```

Comment:
File name should be updateTableSchema_test

Reply:
Will move this test case into the existing update_table_schema_test.go